PatreonRSS.php

```diff
@@ -173,19 +193,24 @@ protected function printRssItem($item)
     {
         echo '<item>';
         echo '<title>';
-        echo htmlspecialchars($item['title']);
+        if(isset($item['title']))
+            echo htmlspecialchars($item['title']);
         echo '</title>';
         echo '<description>';
-        echo htmlspecialchars($item['content']);
+        if(isset($item['content']))
+            echo htmlspecialchars($item['content']);
         echo '</description>';
         echo '<link>';
-        echo htmlspecialchars($item['url']);
+        if(isset($item['url']))
+            echo htmlspecialchars($item['url']);
```

Comment:
An RSS feed item without a URL is pretty useless. I guess this should be checked first and items without a URL be skipped.